packages/components/src/Actions/ActionButton/ActionButton.test.js

```diff
@@ -110,4 +111,12 @@ describe('Action', () => {
 		const wrapper = shallow(<ActionButton available={false} />);
 		expect(wrapper.type()).toBe(null);
 	});
+
+	it('should render action with overlay provider', () => {
+		const actionTest = <Button>Test button popover</Button>;
+		const wrapper = shallow(
+			<ActionButton available={false} overlayComponent={actionTest} overlayPlacement="top" />,
```

Comment:
this test may create miss understanding of the feature.
Please provide a test overlay which is not a Button.
It can be a paragraph and call it 
```
const overlayContent = <p>Hello I'm a overlay</p>
const wrapper = shallow(<ActionButton available={false} overlayComponent={overlayContent} overlayPlacement="top" />);